web/views.py

```diff
@@ -18,6 +20,14 @@ def members(request):
 	members_list = UserProfile.objects.filter(active=True).order_by('fullname')
 	return render(request, 'pages/members.html',{'members_list': members_list})
 
+def member_page(request,user_id):
+	user = get_object_or_404(UserProfile, id=user_id)
```

Comment:
I think there is no understanding in the group what this line do. 

So if there is no user, there will be a 404 error raised, and it will use a 404 template... oh wait, it can't, there is no 404 template defined... So everything will just fall apart.

When there is no users the model will return empty dict, which is probably nicer than 404 at this stage of templates.